tests/endpoints/collection_management.rs

```diff
@@ -0,0 +1,4 @@
+#[test]
+fn collection_management_test() {
```

Comment:
Shouldn't these get implemented as well?

Reply:
These functionalities are not implemented in the testing framework. As far as I know, they were also not available in the older framework.

Reply:
Makes sense